examples/ode_models/ODE_Lotka_Volterra_multiple_ways.myst.md

```diff
@@ -0,0 +1,963 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"tags": []}
+
+(ODE_Lotka_Volterra_fit_multiple_ways)= 
+# ODE Lotka-Volterra With Bayesian Inference in Multiple Ways
+
+:::{post} January 10, 2023
+:tags: ODE, pytensor op, gradient-free inference, pytensor scan
+:category: intermediate, how-to
+:author: [Greg Brunkhorst]{https://github.com/gbrunkhorst}
+:::
+
+```{code-cell} ipython3
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pymc as pm
+import pytensor
+import pytensor.tensor as pt
+
+from pymc.ode import DifferentialEquation
+from pytensor.compile.ops import as_op
+from scipy.integrate import odeint
+from scipy.optimize import least_squares
+
+print(f"Running on PyMC v{pm.__version__}")
+```
+
+```{code-cell} ipython3
+%load_ext watermark
+az.style.use("arviz-darkgrid")
+rng = np.random.default_rng(1234)
+```
+
+## Purpose
+The purpose of this notebook is to demonstrate how to perform Bayesian inference on a system of ordinary differential equations (ODEs), both with and without gradients.  The accuracy and efficiency of different samplers are compared.    
+### Outline
```

Comment:
I would delete this section or at most convert it to a single paragraph outlining the approach in text. See the preview: https://pymcio--478.org.readthedocs.build/projects/examples/en/478/ode_models/ODE_Lotka_Volterra_multiple_ways.html#outline. The in-page table of contents is present in the right hand sidebar (or in a folded box on mobile) and allows navigating to the actual sections. I find the whole expanded outline here without links a bit confusing.

Reply:
Good call - revised per comment.